library/src/test/java/com/shopapp/shopify/api/adapter/CountryListAdapterTest.kt

```diff
@@ -0,0 +1,26 @@
+package com.shopapp.shopify.api.adapter
+
+import com.shopapp.shopify.JodaTimeAndroidRule
+import com.shopapp.shopify.StorefrontMockInstantiator
+import org.junit.Assert.assertEquals
+import org.junit.Assert.assertNotNull
+import org.junit.Rule
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.mockito.junit.MockitoJUnitRunner
+
+@RunWith(MockitoJUnitRunner::class)
+class CountryListAdapterTest {
```

Comment:
Add test for second condition in `countries?.map { CountryAdapter.adapt(it) } ?: listOf()`